python/ray/_private/worker.py

```diff
@@ -516,6 +518,24 @@ def current_task_id(self):
     def current_node_id(self):
         return self.core_worker.get_current_node_id()
 
+    def get_driver_node_id(self, timeout=None) -> ray.NodeID:
+        """
+        Get the driver node id via Job Info. Populates the cached driver node id if
+        found. The fetching can be slow if there are too many jobs.
+
+        @raises Exception if the gcs client times out.
```

Comment:
Can you use the consistent docstring format as other functions here?

```
Args:

Raises:

Returns:
```

Reply:
done